tests/test_int64_indptr.py

```
@@ -0,0 +1,932 @@
+            A.indptr,
+            A.indices,
+            A.data,
+            sums_cells=sums_cells,
+            sums_genes=sums_genes,
+            cell_ex=cell_ex,
+            gene_ex=gene_ex,
+            n_cells=N_ROWS,
+            stream=_stream(),
+        )
+        outs[idx_dtype] = (sums_cells, sums_genes, cell_ex, gene_ex)
+    for a, b in zip(outs[np.int32], outs[np.int64]):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)